external-crates/move/crates/move-ir-to-bytecode-syntax/src/syntax.rs

```diff
@@ -1789,6 +1809,14 @@ fn parse_variant_decl(
 // }
 
 fn parse_module_ident(tokens: &mut Lexer) -> Result<ModuleIdent, ParseError<Loc, anyhow::Error>> {
+    if tokens.peek() == Tok::DotNameValue {
+        let start_loc = current_token_loc(tokens);
+        let module_dot_name = parse_dot_name(tokens)?;
+        let v: Vec<&str> = module_dot_name.split('.').collect();
+        assert!(v.len() == 2);
```

Comment:
Assert instead of a parse error?

Reply:
I think this is an invariant of the token `DotNameValue`

Reply:
Might keep this as-is -- this is an invariant as Todd mentions over `DotNameValue` and this follows other places in this code where we have the same type of assert.